code/controllers/master.dm

```diff
@@ -157,6 +159,20 @@ ADMIN_VERB(cmd_controller_view_ui, R_SERVER|R_DEBUG, "Controller Overview", "Vie
 
 	var/list/subsystem_data = list()
 	for(var/datum/controller/subsystem/subsystem as anything in subsystems)
+		var/list/rolling_usage = subsystem.rolling_usage
+		// First, trim er up
+		var/cut_to = 0
+		while(cut_to + 1 <= length(rolling_usage) && text2num(rolling_usage[cut_to + 1]) < (world.time - rolling_usage_length) / world.tick_lag)
```

Comment:
DS2TICK is already a macro and makes the logic easier to understand.

Reply:
o7